content/en/graphing/_index.md

```diff
@@ -9,128 +9,27 @@ aliases:
-
-{{< img src="graphing/index/arithmetic_3.png" alt="Arithmetic 3" responsive="true" style="width:75%;" >}}
-
-To display only your formula, un-check your metrics **a** and **b**:
-
-{{< img src="graphing/index/arithmetic_3_bis.png" alt="Arithmetic 3 bis" responsive="true" style="width:75%;" >}}
-
-**Note**: Formulas are not lettered. Arithmetic cannot be done between formulas.
-
-## Create a title
-
-If you don't enter a title, one is automatically generated based on your selections. However, it is recommended that you create a title that aptly describes the purpose of the graph. 
-
-## Save
-
-Click **Done** to save your work and exit the editor. You can always come back to the editor to change the graph. If you make changes you don't want to save, click **Cancel**.
+{{< whatsnext desc="This section includes the following topics:" >}}
+    {{< nextlink href="/graphing/using_graphs/" >}}<u>Using graphs</u>: Whether using metrics, monitors, dashboards, or notebooks, all graphs have basic functionality in common - start here for best practices for setting up graphs.{{< /nextlink >}}
+    {{< nextlink href="/graphing/dashboards" >}}<u>Dashboards</u>: A dashboard is Datadogs tool that visually tracks, analyzes, and displays key performance metrics to monitor the health of your infrastructure. Learn more about using Screenboards, Timeboards, optimizing those dashboards with template variables, and sharing your graphs to people outside of your Datadog account.{{< /nextlink >}}
+    {{< nextlink href="/graphing/metrics/" >}}<u>Metrics</u>: When you connect an integration to Datadog, the integration sends metrics about your system back to Datadog. Once Datadog receives these metrics, you can build graphs to explore the data. The Metrics Explorerlets you search for specific metrics and then create graphs based on your search parameters, and the Metrics Summary shows all of the metrics coming into Datadog.{{< /nextlink >}}
+    {{< nextlink href="/graphing/notebooks" >}}<u>Notebooks</u>: Notebooks combine graphs and text in a linear, cell-based format. They’re designed to help you explore and share stories with your data.{{< /nextlink >}}
+    {{< nextlink href="/graphing/event_stream/" >}}<u>Event Stream</u>: The event stream is a display of the most recent events in your infrastructure.{{< /nextlink >}}
+    {{< nextlink href="/graphing/infrastructure/" >}}<u>Infrastructure</u>: The Infrastructure list page shows all hosts monitored by your Datadog application. Host Maps visualize hosts together on one screen, with metrics made comprehensible via color and shape, and the container map and live containers shows you the health of your containers. Process Monitoring allows for real-time visibility of the most granular elements in a deployment. Cloud functions are a service that runs your code in response to events and automatically manage the underlying compute resources for you.{{< /nextlink >}}
```

Comment:
```suggestion
    {{< nextlink href="/graphing/infrastructure/" >}}<u>Infrastructure</u>: The Infrastructure list page shows all hosts monitored by your Datadog application, including Host and Container Maps, Process Monitoring, and Cloud Functions.{{< /nextlink >}}
```

I feel as though going into the details of what each of these are is better on the actual pages.